www/src/components/Menu.tsx

```diff
@@ -35,15 +35,15 @@ const menuQuery = graphql`
         }
         frontmatter: { type: {} }
       }
-      sort: { fields: frontmatter___title }
+      sort: { frontmatter: {title: ASC} }
     ) {
-      categories: group(field: frontmatter___categories) {
+      categories: group(field: {frontmatter: {categories: SELECT}}) {
         nodes {
           ...ComponentPage
         }
         fieldValue
       }
-      types: group(field: frontmatter___type) {
+      types: group(field: {frontmatter: {type: SELECT}}) {
```

Comment:
I'm not sure if these are the specific changes causing the change, but I noticed on the PR deploy preview contains additional links in the footer menu compared to the v22 docs site.

The PR deploy preview seems to include static content pages instead of the intended exports (i.e., components, hooks, etc.). For example, the footer menu now contains things like "Brand-Icons", "Component generation", "Getting started", etc.).

Is this intentional?

Also worth noting, clicking at least the "CSS Utilities" in the footer menu leads to a 404 not found page ([link](https://deploy-preview-3367--paragon-openedx-v22.netlify.app/css-utilities-content/)).

Reply:
Fixed!